src/OpenLoco/Math/Vector.hpp

```diff
@@ -2,7 +2,7 @@
 
 #include <cstdint>
 #include <cstdio>
-#include <string>
+#include <numeric>
```

Comment:
Ubuntu CI is failing; apparently because of `std::abs` not being defined. Could you include the `cmath` header here?

Reply:
Doh i looked at https://en.cppreference.com/w/cpp/numeric/math/abs and saw the url has numeric and just thought that was the header.

Reply:
Fixed